### PR TITLE
fix(ts): does not mark ts release as latest in github

### PR DESCRIPTION
**Pull request by ygrishajev**

### CODE REVIEW

File: script/release-ts.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```
Comment (ygrishajev) [inline script/release-ts.sh:1]: this script does the following:
1) gets current latest release
2) runs semantic release which creates a new release for ts and marks it as latest (github default behaviour)
3) marks back prev latest release as current latest

**Comment by ygrishajev** [comment]
@baktun14 FYI https://github.com/akash-network/akash-api/pull/146/commits/b3ae94e0109fa1747557733a5ccd4bd2b97ec067 :)

File: ts/.releaserc [hunk 2]
```diff
@@ -21,7 +21,7 @@
       }
     ]
   ],
-  "preset": "angular",
+  "preset": "conventionalcommits",
```
Comment (ygrishajev) [inline ts/.releaserc:24]: @baktun14 FYI :)

**Review by baktun14 — APPROVED** [review]

File: script/release-ts.sh (hunk 1 above)
Comment (troian) [reply script/release-ts.sh:1]: lets get this script under top level `script` dir

File: script/release-ts.sh [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+API_URL="https://api.github.com"
+
```
Comment (troian) [inline script/release-ts.sh:4]: add check if `$GITHUB_TOKEN"` is set and error if not

Comment (troian) [reply script/release-ts.sh:4]: same for `$GITHUB_ACTION_REPOSITORY` and `"$AKASH_TS_ROOT"`

File: script/release-ts.sh (hunk 1 above)
Comment (ygrishajev) [reply script/release-ts.sh:1]: it is there already

Comment (troian) [reply script/release-ts.sh:1]: blind

File: script/release-ts.sh (hunk 3 above)
Comment (ygrishajev) [reply script/release-ts.sh:18]: done

File: script/release-ts.sh [hunk 4]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+if [ -z "$GITHUB_REPOSITORY" ]; then
```
Comment (troian) [inline script/release-ts.sh:3]: `GITHUB_REPOSITORY=${GITHUB_REPOSITORY:-$GITHUB_ACTION_REPOSITORY}`

File: script/release-ts.sh [hunk 5]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```
Comment (troian) [inline script/release-ts.sh:1]: `/usr/bin/env bash`

**Review by troian — APPROVED** [review]

**Pull request merged**